Homepage Update Fails with pnpm

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Homepage

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/homepage.sh)"

### 📝 Provide a clear and concise description of the issue.

I'm getting the following error when running the installation script to update Homapge to the latest version.



### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Step 1 - Ran the following command from within the lxc: `bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/homepage.sh)"`

Step 2 - Select the update function. I tried both options 1 - Yes (Silent Mode) and 2 - Yes (Verbose Mode)

Script error out.

### ❌ Paste the full error output (if available).

⠇ Updating Homepage to v0.10.9 (Patience)Latest version:     1.0.30001702
Updating caniuse-lite version
$ pnpm up caniuse-lite
⠋ Updating Homepage to v0.10.9 (Patience) ERR_PNPM_UNEXPECTED_STORE  Unexpected store location

The dependencies at "/opt/homepage/node_modules" are currently linked from the store at "/root/.local/share/pnpm/store/v3".

pnpm now wants to use the store at "/opt/homepage/.pnpm-store/v3" to link dependencies.

If you want to use the new store location, reinstall your dependencies with "pnpm install".

You may change the global store location by running "pnpm config set store-dir <dir> --global".
(This error may happen if the node_modules was installed with a different major version of pnpm)

Error: Command failed: pnpm up caniuse-lite
    at genericNodeError (node:internal/errors:984:15)
    at wrappedFn (node:internal/errors:538:14)
    at checkExecSyncError (node:child_process:891:11)
    at execSync (node:child_process:963:15)
    at updateWith (/root/.npm/_npx/2662cbd72e57abb6/node_modules/update-browserslist-db/index.js:263:5)
    at updateDB (/root/.npm/_npx/2662cbd72e57abb6/node_modules/update-browserslist-db/index.js:298:5)
    at Object.<anonymous> (/root/.npm/_npx/2662cbd72e57abb6/node_modules/update-browserslist-db/cli.js:34:5)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)

Problem with `pnpm up caniuse-lite` call. Run it manually.
⠙ Updating Homepage to v0.10.9 (Patience)
[ERROR] in line 48: exit code 0: while executing command $STD npx --yes update-browserslist-db@latest

### 🖼️ Additional context (optional).

I got it to work by executing the script steps manually and by changing the order of execution of the steps.

In the script `ProxmoxVE/ct/homepage.sh` I switch the execution order of the following lines:

 line 48.      $STD npx --yes update-browserslist-db@latest
 line 49.      $STD pnpm install

first ran `pnpm install` and then `npx --yes update-browserslist-db@latest`

every other step ran successfully and Homepage has been updated to the latest version.

--- a/ct/homepage.sh
+++ b/ct/homepage.sh
@@ -45,8 +45,8 @@ function update_script() {
     cp -r homepage-${RELEASE}/* /opt/homepage/
     rm -rf homepage-${RELEASE}
     cd /opt/homepage
-    $STD npx --yes update-browserslist-db@latest
     $STD pnpm install
+    $STD npx --yes update-browserslist-db@latest
     export NEXT_PUBLIC_VERSION="v$RELEASE"
     export NEXT_PUBLIC_REVISION="source"
     $STD pnpm build

